manifests/agent.pp

```diff
@@ -12,6 +12,8 @@
   $config_group                 = 'root',
   $config_mode                  = '0644',
   $env                          = $::env,
+  $puppet_run_interval          = '1h',
+  $puppet_reports               = 'true',
```

Comment:
This should be singular `report` since it manages the `report` configuration option and not the `reports` configuration option. Suggest putting this into a separate pull request along with spec tests for a quick merge.